Automattic-Tracks-iOSTests/Mock Data/explat-assignments.json

```diff
@@ -0,0 +1,6 @@
+{
+    "variations": {
+        "experiment": "control"
```

Comment:
Explat will never return the name of the control variation, For control, it always returns `null` and/or it won't exist in the response. It only returns the name of treatment variations. Right now, we only support a singular treatment, but multi-variate experiments are in the works.